rotkehlchen/tests/unit/globaldb/test_globaldb_consistency.py

```diff
@@ -116,10 +115,10 @@ def test_asset_updates_consistency_with_packaged_db(
         globaldb.conn.read_ctx() as old_db_cursor,
         globaldb.packaged_db_conn().read_ctx() as packaged_db_cursor,
     ):
-        # Assets version here is 32 because:
-        # - Global DB v9->v10 includes breaking schema changes
-        # - Before such changes, we pull all compatible asset updates up to v32 (max compatible)
-        assert old_db_cursor.execute("SELECT value FROM settings WHERE name='assets_version'").fetchone()[0] == '32'  # noqa: E501
+        # Assets version here is 36 because:
+        # - Global DB v9->v10 & v12 -> v13 includes breaking schema changes
+        # - Before such changes, we pull all compatible asset updates up to v32 and then v36 (max compatible)  # noqa: E501
```

Comment:
Mention that this happens as part of `apply_pending_compatible_updates` that executes when creating the db object before and at this point we are sure that assets updates up until 36 are applied.

Also I believe that calling it old_db_cursor is confusing since it is the actual db that you are testing